### ISSUE TITLE
Don't allow eval just for OCRAD.js

### ISSUE THREAD

**Issue by siphomateke** [siphomateke/zra-helper#102]

**Comment by hbzhang3** [comment]
exactly, @siphomateke did you find any alternatives for this? I tried Tesseract, it is much slower than ocrad.

**Comment by siphomateke** [comment]
@hbzhang3 Hey, I haven't worked on this project in over 3 years, so I'm afraid I can't help much. That said, I switched to using a more advanced OCR model than Ocrad.js, which you can see here https://github.com/siphomateke/zra-captcha-solver.
